modules/gapi/src/streaming/onevpl/accelerators/surface/dx11_frame_adapter.cpp

```diff
@@ -114,17 +114,21 @@ MediaFrame::View VPLMediaFrameDX11Adapter::access(MediaFrame::Access mode) {
     }
 }
 
-cv::util::any VPLMediaFrameDX11Adapter::blobParams() const {
-    /*GAPI_Assert(false && "VPLMediaFrameDX11Adapter::blobParams() is not fully integrated"
-                         "in OpenVINO InferenceEngine and would be temporary disable.");*/
-#ifdef HAVE_INF_ENGINE
+mfxHDLPair VPLMediaFrameDX11Adapter::getHandle() const {
```

Comment:
it should be 'private'  if possible

Reply:
I don't think it's possible. Also this adapter itself is not public

Reply:
I propose to emphasize that this temporary hack by using 'friend'. It will be temporary, of course, until a correct solution implemented.

Feel free to resolve this thread if declined